build.gradle

```diff
@@ -16,7 +16,6 @@ apply plugin: 'idea'
 apply plugin: 'java'
 apply plugin: 'maven'
 apply plugin: 'com.google.protobuf'
-apply plugin: 'checkstyle'
```

Comment:
Didn't even use that. I think it came from the rebasing. I only changed (and added and committed) Map.java and CatchablePokemon.java.